### PR TITLE
Adding access key auth support for openstack V3

### PR DESCRIPTION
**Pull request by DImuthuUpe**
Please refer to the discussion thread [1]

[1] https://lists.apache.org/thread/2p7qc20rn19r3yhsfd63p3o3h587f7oh

### CODE REVIEW

**Comment by smarru** [comment]
@gaul drawing your attention, can you please review and provide us feedback, we would like to use this in Apache Airavata

File: apis/openstack-keystone/src/main/java/org/jclouds/openstack/keystone/v3/auth/V3AuthenticationApi.java [hunk 1]
```diff
@@ -57,4 +60,10 @@ public interface V3AuthenticationApi extends AuthenticationApi, Closeable {
    @Override
    Token authenticateToken(TenantOrDomainAndCredentials<TokenCredentials> credentials);
 
+   @Named("token:create")
+   @POST
+   @ResponseParser(ParseTokenFromHttpResponse.class)
+   @MapBinder(BindAccessKeyAuthToJsonPayload.class)
+   @Override
+   AuthInfo authenticateAccessKey(TenantOrDomainAndCredentials<ApiAccessKeyCredentials> credentials);
```
Comment (nacx) [inline apis/openstack-keystone/src/main/java/org/jclouds/openstack/keystone/v3/auth/V3AuthenticationApi.java:68]: Please add the corresponding unit tests here: https://github.com/apache/jclouds/blob/master/apis/openstack-keystone/src/test/java/org/jclouds/openstack/keystone/v3/auth/V3AuthenticationApiMockTest.java

And the live test as well here: 
https://github.com/apache/jclouds/blob/master/apis/openstack-keystone/src/test/java/org/jclouds/openstack/keystone/v3/auth/V3AuthenticationApiLiveTest.java

And include in a comment in the PR the output of running the live tests.

**Review by nacx — COMMENTED** [review]
Thanks!

**Review by gaul — CHANGES_REQUESTED** [review]
@DImuthuUpe Can you add tests?

**Comment by gaul** [comment]
@DImuthuUpe Can we move forward with this or should I close it?

**Comment by gaul** [comment]
Please re-open if this is still relevant.

**Comment by edipdemirbilek** [comment]
@gaul I have run into the same issue. i wonder is this was ever fixed.

**Pull request closed without merging**